dz7.py

```diff
@@ -0,0 +1,42 @@
+"""Convert a non-negative integer num toits English words representation."""
+
+
+# Function to receive numbers and convert to words
+def number_to_words(num):
+    if 100 <= num <= 999:
+        # Списки слов для единиц, десятков и сотен
+        ones = ['', 'One', 'Two', 'Three', 'Four',
+                'Five', 'Six', 'Seven', 'Eight', 'Nine']
+        tens = ['', '', 'Twenty', 'Thirty', 'Forty',
+                'Fifty', 'Sixty', 'Seventy', 'Eighty', 'Ninety']
+        teens = ['Ten', 'Eleven', 'Twelve', 'Thirteen', 'Fourteen',
+                 'Fifteen', 'Sixteen', 'Seventeen', 'Eighteen', 'Nineteen']
+
+        # Разделение числа на сотни, десятки и единицы
```

Comment:
Comments on Russian acceptable , but the best option - leave comments on English